### PR TITLE
Next bunch of design iterations

### PR DESCRIPTION
**Pull request by pacxiu**
# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️ 
- added `actionOption` collapsed design to Open/Manage flows
- added `After Pills` for all flows (Details Boxes and Summary), coloring based on Collateralization Ratio (feedback from design team)
- added "?" for all flows + mocked Modal
image

image
- switch to multiply/borrow is moved to bottom of Open flows
- added animation for open and manage regular vault flows - Lottie file was provided by Hazel (best to see it in storybook)

- added `liquidationPriceCurrentPriceDifference` for `% below current price` for liquidation price card
- added `collateralizationRatioNextPrice` calculation
  
## How to test 🧪
- inspect storybook and or deployment preview to see changes
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

File: features/openMultiplyVault/components/OpenMultiplyVaultDetails.tsx [hunk 1]
```diff
@@ -1,56 +1,159 @@
...
-              {formatPercent(afterCollateralizationRatio.times(100), {
-                precision: 2,
-                roundMode: BigNumber.ROUND_DOWN,
-              })}
-            </Text>
-            <Text as="span" sx={{ color: 'text.subtitle' }}>
-              {` ${t('system.collateralization-ratio')}`}
-            </Text>
+            {/* TO DO calculations */}
+            {formatAmount(new BigNumber(1), 'DAI')}
```
Comment (paszkowskiDamian) [inline features/openMultiplyVault/components/OpenMultiplyVaultDetails.tsx:38]: Initial debt will always be `0` ;)

File: features/openMultiplyVault/components/OpenMultiplyVaultDetails.tsx [hunk 2]
```diff
@@ -1,56 +1,159 @@
...
+            {` DAI`}
           </>
         }
+        valueAfter={
+          showAfterPill && (
+            <>
+              {/* TO DO calculations */}
+              {formatAmount(new BigNumber(2), 'DAI')}
```
Comment (paszkowskiDamian) [inline features/openMultiplyVault/components/OpenMultiplyVaultDetails.tsx:46]: Isn't `afterOutstandingDebt` already available on feature branch?

**Comment by paszkowskiDamian** [comment]
image

image


Looks like the there is no `Vault debt | Available to generate | Available to withdraw` section on the designs for multiply

**Comment by pacxiu** [comment]
> image
> 
> image
> 
> ~Looks like the there is no `Vault debt | Available to generate | Available to withdraw` section on the designs for multiply~
> 
> Sorry, this need to be verified with Design team. This element appears on some screens.

Idea is it should be also avaialble (Maybe we will have different content for Multiply)

**Pull request merged**